clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10264,6 +10264,10 @@ def warn_depr_array_comparison : Warning<
   "to compare array addresses, use unary '+' to decay operands to pointers">,
   InGroup<DeprecatedArrayCompare>;
 
+def warn_array_comparison : Warning<
+  "comparison between two arrays compare their addresses and will be deprecated in c++20; "
+  "to compare array addresses, use unary '+' to decay operands to pointers">;
```

Comment:
Did this lose its group?  It still needs one.  Also, I thought someone had suggested doing it inline? Diag message seems acceptable enough to me.

Reply:
I think i misunderstood inlining diagnostic here, can you give me example about how inline it

Reply:
He means to inline the diagnostic group, so instead of `InGroup<SomeName>` you can do: `InGroup<DiagnosticGroup<"Group name">>`

(I think you might have to grep here for proper spelling).

Reply:
Cool i found `InGroup<DiagGroup<....>>` thanks, for release note i think `array-comparion` i think it be under New Compiler Flags